docs/pages/base-ui/api/number-input.json

```diff
@@ -75,7 +75,12 @@
   ],
   "classes": {
     "classes": ["disabled", "error", "focused", "formControl", "readOnly"],
-    "globalClasses": { "focused": "Mui-focused", "disabled": "Mui-disabled", "error": "Mui-error" }
+    "globalClasses": {
```

Comment:
`docs:api` has generated this change and [this change](https://github.com/mui/material-ui/pull/38424/files#diff-08cd52e0dcc716904961fdcc988e90d41c2d6e7680251e42b2f16fc66ccdd064). Is this a bug in `script`?

Reply:
When the [number input classes were added](https://github.com/mui/material-ui/commit/f95ad25a85f082ab3d3a30a456809f7c712b7026#diff-39e4fe7cf63a743736388a7ce9e48fe9c6495643bb0834aa3e18a28704a5fd96), we missed running `yarn docs:api`, so it was [committed after](https://github.com/mui/material-ui/commit/0aa7db4da1158b8c1efe3951d57ced81cea517ac). The `select-class` branch was probably created in between these changes.

This change should disappear if we merge `master` into the `select-class` branch. May I ask you to do so?

Reply:
fixed.